CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.2
+- Remove Encryptable::OSESecret like in Cryptopus 5.0 
```

Comment:
Support for Openshift/k8s Secrets has been dropped in Cryptopus 5.0 - also removed from CLI